scene/gui/text_edit.cpp

```diff
@@ -4469,15 +4469,25 @@ int TextEdit::get_visible_line_count() const {
 int TextEdit::get_visible_line_count_in_range(int p_from_line, int p_to_line) const {
 	ERR_FAIL_INDEX_V(p_from_line, text.size(), 0);
 	ERR_FAIL_INDEX_V(p_to_line, text.size(), 0);
-	ERR_FAIL_COND_V(p_from_line > p_to_line, 0);
+
+	int smaller;
+	int larger;
+
+	if (p_from_line > p_to_line) {
+		smaller = p_to_line;
+		larger = p_from_line;
+	} else {
+		smaller = p_from_line;
+		larger = p_to_line;
+	}
```

Comment:
```suggestion
	if (p_from_line > p_to_line) {
		SWAP(p_from_line, p_to_line);
	}
```

Reply:
Huh. Never knew about that macro. Will do.